<?xml version="1.0"?>

<launch>
    <arg name="rviz" default="true"/>

    <arg name="robot_ip" default="10.42.0.238"/>
    
    <include file="$(find ur_modern_driver)/launch/ur10_bringup.launch">
        <arg name="robot_ip" value="$(arg robot_ip)"/>
    </include>

    <include file="$(find i3dr_phobos)/launch/phobos.launch">
        <arg name="rviz" value="false"/>
    </include>

    <node name="i3dr_hand_eye" pkg="i3dr_ur10" type="i3dr_hand_eye" output="screen">
        <param name="camera_namespace" type="string" value="/phobos"/>
        <param name="world_tf" type="string" value="/world"/>
        <param name="arm_tooltip_tf" type="string" value="/tool0"/>
    </node>

    <!-- DISPLAY -->
    <!-- run rviz with config file -->
    <node if="$(arg rviz)" pkg="rviz" type="rviz" name="rivz" 
        args="-d $(find i3dr_ur10)/rviz/ur10_phobos_nuclear.rviz"/>
</launch>